srv/modules/runners/changed.py

```diff
@@ -129,8 +160,8 @@ def __init__(self, **kwargs):
         self.base_dir = '/srv/salt/ceph/configuration/files/'
         self.checksum_dir = self.base_dir + 'ceph.conf.checksum/'
         self.checksum_file = self.checksum_dir + self.role.conf_filename + self.role.conf_extension
-        log.debug("dependencies of role {}: {}".format(self.role.name,
-                                                       self.role.dependencies_unwrapped()))
+        log.debug("dependencies of role {}: {}".format(
+            self.role.name, self.role.dependencies_unwrapped()))
```

Comment:
Change in whitespace to stay under char limit per line?

Reply:
I reverted the style changes